content/streamlit-cloud/deploy-your-app/index.md

```diff
@@ -55,15 +55,15 @@ Although you can deploy multiple apps from the same repository, there can be onl
 
 ## Advanced settings for deployment
 
-3. (Optional) If you are connecting to a data source or want to specify the Python version for your app, you can do that by clicking "**Advanced settings**" before you deploy the app. Learn more about [Secrets management](/streamlit-community-cloud/deploy-your-app/secrets-management).
+<Note>
 
-   ![Advanced settings for deploying your app](/images/streamlit-community-cloud/deploy-an-app-advanced.png)
```

Comment:
> Streamlit Community Cloud supports all released [versions of Python](https://devguide.python.org/versions/) that are still receiving security updates.

Nit: recommend to put the link on "still receiving security updates", or else on the whole phrase, for emphasis.

As-is, I might click the link without reading the security updates part and see all the versions from 2.6+ and get confused.

Reply:
Updated.